sdg/components.py

```diff
@@ -19,7 +19,7 @@ def git_clone_op(
             f"git clone {repo_url} {taxonomy.path} && cd {taxonomy.path} && "
             + f'if [ ! -z "{repo_branch}" ]; then '
             + f"git fetch origin {repo_branch} && git checkout {repo_branch}; "
-            + f'elif [ ! -z "{repo_pr}" ]; then '
+            + f'elif [ ! -z "{repo_pr}" and {repo_pr} -gt 0 ]; then '
```

Comment:
```suggestion
            + f'elif [ ! -z "{repo_pr}" && {repo_pr} -gt 0 ]; then '
```
`and`? in bash? :)

Reply:
Also, a good opportunity to switch from `! -z` with `-n` :)

Reply:
c16d456ae504da7a5c53fb8573a784fb1e5be6c8